crates/sdk/src/callbacks.rs

```diff
@@ -930,8 +939,6 @@ impl CredentialStore {
             );
         }
 
-        self.callbacks.invoke((creds.clone(), address), state);
```

Comment:
Confirming that this just moves this call to after some log tracing?

Reply:
Oh, yeah, this was an instinctive edit to remove half a clone. Moving the callbacks.invoke to the end means you don't need to clone at all if you already have stored creds.